Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Reflection.PortableExecutable;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static float globalHeight = 0f;
+        public static float globalWeight = 0f;
```

Comment:
There is no point keeping it static somewhere at the top. Keep variables close where you use them. Also discard the naming pattern where acccess modifier is involved.